### ISSUE TITLE
Class extends value undefined is not a constructor or null

### ISSUE THREAD

**Issue by karneaud** [interfaced/zombiebox#2]
In trying to `zb run` a project using the instructions in the[ documentation](https://zombiebox.tv/doc.html) I ran into the subject error.

My package.json

```
{
  "name": "zombieox-eatahfood",
  "version": "0.1.0",
  "dependencies": {
    "zombiebox": "1.0.0-alpha4",
    "zombiebox-extension-ui": "0.5.1"
  },
  "scripts": {
    "start": "zombiebox run",
    "test": "zombiebox build pc --config config.js --config configs/check.js",
    "build.prod.pc": "zombiebox build pc --config config.js --config configs/prod.js",
    "build.test.pc": "zombiebox build pc --config config.js --config configs/test.js"
  }
}
```

> TypeError: Class extends value undefined is not a constructor or null
>     at Object.<anonymous> (/usr/src/app/node_modules/zombiebox-platform-pc/index.js:16:26)
>     at Module._compile (internal/modules/cjs/loader.js:774:30)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
>     at Module.load (internal/modules/cjs/loader.js:641:32)
>     at Function.Module._load (internal/modules/cjs/loader.js:556:12)
>     at Module.require (internal/modules/cjs/loader.js:681:19)
>     at require (internal/modules/cjs/helpers.js:16:16)
>     at Extensions._loadAddon (/usr/src/app/node_modules/zombiebox/lib/extensions.js:147:11)
>     at Extensions._useAddon (/usr/src/app/node_modules/zombiebox/lib/extensions.js:128:25)
>     at Extensions._usePlatform (/usr/src/app/node_modules/zombiebox/lib/extensions.js:108:25)

I ad the following warnings on installation though

> Linking dependencies...
> warning "zombiebox > zombiebox-platform-pc@2.0.0-alpha.3" has incorrect peer dependency "zombiebox@^2.0.0-alpha.5".

**Comment by l1bbcsg** [comment]
ZombieBox `1.0.0-alpha4` is quite old version and the framework was not public at that point yet, looks like it's referring to incorrect npm dist tags for its dependencies.

Since that version we released `1.0.0` – a stable version equivalent to previous releases and `2.0.0` – a new release that we're currently in the process of rolling out.

The documentation at zombiebox.tv is outdated (it's referring to versions event lower than 1.0) and was meant to be taken down. The correct documentation for 2.0.0 is in [README.md](https://github.com/interfaced/zombiebox/blob/master/README.md) file and [docs](https://github.com/interfaced/zombiebox/tree/master/docs) folder.

To solve the issue I'd recommend using either `1.0.0` or `2.0.0` branches, but both have caveats as well: 

1.0 is stable, but its support has ended and its lacking proper public documentation.

2.0 rollout and publishing is taking some time and you being an early adopter are collecting all of the first release errors, apologizes for the inconvenience and thanks for reporting all this. 

We plan to get 2.0 production ready in the coming days.

**Comment by karneaud** [comment]
@l1bbcsg 

Thanks for the update. I managed to update to the _2.0.0 branch_ and with a bit more fiddling got things to launch using the [demo](https://github.com/interfaced/zombiebox-demo)  I am yet to test the "builds" but will notify you when I do with my findings
